.travis.yml

```diff
@@ -1,13 +1,7 @@
 sudo: false
 language: go
 go:
-  - 1.6.3
-addons:
-  apt:
-    sources:
-      - sourceline: 'ppa:masterminds/glide'
-    packages:
-      - glide
+  - 1.8.3
 before_install:
   ## Install App Engine SDK for Go
   - curl -o $HOME/go_appengine_sdk_linux_amd64.zip https://storage.googleapis.com/appengine-sdks/featured/go_appengine_sdk_linux_amd64-1.9.48.zip
```

Comment:
App Engine SDK 1.9.48 (go 1.6) is too old.

Reply:
@minimum2scp Modified `.travis.yml` to use `1.9.61` instead.